pkgs/applications/backup/nis-one-click-backup/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, cmake
+, ninja
+, qt6
+, wrapQtAppsHook
+, extra-cmake-modules
+}:
+
+stdenv.mkDerivation rec {
+  pname = "one-click-backup";
+  version = "1.2.2";
+
+  src = fetchFromGitLab {
+    owner = "dev-nis";
+    repo = "nis-one-click-backup-qt";
+    rev = "9a1ca839ca511d55716ffc8898e5d9ec9449ef18"; #version;
+    hash = "sha256-JbgmfKEpKRj/ph8dkW0IRulmAhk6Dwv/1Z4j3hUgPXk=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+    wrapQtAppsHook
+    extra-cmake-modules
```

Comment:
Just as a note, not a requirement for me: Should perhaps be moved to `buildInputs` (`strictDeps = true` will break this here), but most of Nixpkgs puts it in `nativeBuildInputs` and doesn't enforce `strictDeps` yet… So I guess it's okay to leave it here.

Reply:
Even though you say it's not an requirement, I still implemented your suggestion with commit [6098f80](https://github.com/NixOS/nixpkgs/pull/287497/commits/6098f80dab4a5e5bddae62a9bd10e6ef6b8e2831)